src/watch/watch-proxy.ts

```diff
@@ -31,6 +53,11 @@ async function watchInternal(configs: MaybeArray<RollupOptions>, emitter: Rollup
 			)
 		);
 	}
+	if (!checkWatchConfig(watchOptionsList)) {
+		return error(
+			logInvalidOption('watch', URL_WATCH, 'the input should not be inside the output dir')
```

Comment:
If we throw the error within checkWatchConfig instead, then we can actually print the offending paths, which users might like if they have multiple configs.

Reply:
That makes sense.